src/components/wasd-to-analog2d.js

```diff
@@ -0,0 +1,100 @@
+AFRAME.registerComponent("wasd-to-analog2d", {
```

Comment:
This is pretty ugly but its just pointing out a hole in AFIM, so this is good for now.